### ISSUE TITLE
"Download" page

### ISSUE THREAD

**Issue by OndrejM** [jcpevangelists/javaee.io#34]
New download page in bottom menu under Java EE:
 - link to Java EE SDK
 - link to download page of Glassfish
 - links to download pages of certified servers
  - links to other projects with downloadables (Forge)

**Comment by mikecroft** [comment]
I was going to get started on making a first draft of this page. Looking over the repo it looks like I can just create a `../pages/download.adoc` and `../pages/download.adoc.yaml`?

As for the content, I was going to link to GlassFish at the top as the RI then make a table of available app servers with a bit of detail about each, e.g. vendor, open source/closed source, versions, supported/unsupported etc.

Adding downloads to other projects might be a bit out of scope (bit too much overlap with tools?) though there is the argument for keeping *all* downloads to this page, and reserving other pages for only descriptions. We'll see how it looks.

**Comment by mikecroft** [comment]
Could you assign this issue to me? I don't know if non-collaborators can be assigned, so add me if you need to

**Comment by OndrejM** [comment]
You are right, other downloadables should go into "tools" section, unless somewhat special and essential. The download section is meant to contain essential downloadables that most people need to get started. So far, having JavaEE SDK, GlassFish and info about other runtimes should be enough. Later, I would also put here some getting started artefacts, like project templates.

**Comment by OndrejM** [comment]
It's not possible to assign the issue to you, you have to get wors write access to the repo I guess. @tveronezi, @jcpevangelists/admins ?
